site/_docs/algebra.md

```diff
@@ -309,6 +299,19 @@ LogicalRepeatUnion(all=[true])
         LogicalTableScan(table=[[aux]])
 {% endhighlight %}
 
+Note that currently there is no support for recursive queries in the SQL layer yet.
+At some point in the future, a recursive SQL query may look like this
+(example only for illustrative purposes, the `RECURSIVE` keyword is not supported yet):
+
+{% highlight sql %}
+WITH RECURSIVE aux(i) AS (
+  VALUES (1)
+  UNION ALL
+  SELECT i+1 FROM aux WHERE i < 10
+)
+SELECT * FROM aux
+{% endhighlight %}
+
```

Comment:
Maybe we can keep the note and here just write: "The SQL example above is only for illustrative purposes (`RECURSIVE` keyword not supported)."